### PR TITLE
List should load names rather than ids - new

### PR DESCRIPTION
**Pull request by AvihayMReblaze**

### CODE REVIEW

File: src/views/DynamicRulesList.vue [hunk 1]
```diff
@@ -111,14 +111,13 @@ export default defineComponent({
         {
           title: 'Action',
```
Comment (Aviv-Galmidi) [inline src/views/DynamicRulesList.vue:112]:
```suggestion
          title: 'Custom Response',
```

File: src/views/DocumentList.vue [hunk 2]
```diff
@@ -188,13 +203,552 @@ export default defineComponent({
       return `configs/${this.selectedBranch}/d/${this.selectedDocType}/v/`
     },
 
+    columnOptionMap() {
```
Comment (Aviv-Galmidi) [inline src/views/DocumentList.vue:206]: We can also delete `documentListConst.ts`

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/views/DynamicRulesList.vue (hunk 1 above)
Comment (AvihayMReblaze) [reply src/views/DynamicRulesList.vue:112]: Done

File: src/views/DocumentList.vue (hunk 2 above)
Comment (AvihayMReblaze) [reply src/views/DocumentList.vue:206]: Done

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**